.gitignore

```diff
@@ -3,3 +3,28 @@ build/
 
 .vagrant/
 *.swp
+
+# Common IntelliJ Platform excludes
+
+# User specific
+cmake-build-*
+**/.idea/**/workspace.xml
+**/.idea/**/tasks.xml
+**/.idea/shelf/*
+**/.idea/dictionaries
+**/.idea/httpRequests/
+
+# Sensitive or high-churn files
+**/.idea/**/dataSources/
+**/.idea/**/dataSources.ids
+**/.idea/**/dataSources.xml
+**/.idea/**/dataSources.local.xml
+**/.idea/**/sqlDataSources.xml
+**/.idea/**/dynamic.xml
+
+# CLion
+# CLion auto-generates .iml files, and contentModel.xml
+.idea/
```

Comment:
Simply keep this line
```
.idea/
```
the remaining part of this `.gitignore` patch it's just noise IMHO
ref: https://git-scm.com/docs/gitignore

question: do you have some .idea directory at other place than the repository root directory ?

Reply:
at the moment I don't have .idea at other places